src/server/game/Events/GameEventMgr.cpp

```diff
@@ -127,10 +128,10 @@ bool GameEventMgr::StartEvent(uint16 event_id, bool overwrite)
             if (data.end <= data.start)
                 data.end = data.start + data.length;
         }
-#ifdef ELUNA
+
         if (IsActiveEvent(event_id))
-            sEluna->OnGameEventStart(event_id);
```

Comment:
do not delete sEluna stuff ?

Reply:
Looks fine to me. He simply replaced the eluna hooks with C++ hooks. Then the C++ hooks call the eluna hooks.

Reply:
ok thank you mate!